app/bundles/LeadBundle/Event/DuplicateContactsEvent.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * @copyright   2019 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\LeadBundle\Event;
+
+class DuplicateContactsEvent extends \Mautic\ChannelBundle\Event\ChannelEvent
+{
+    /**
+     * @var array
+     */
+    private $fields;
+
+    /**
+     * @var array
+     */
+    private $duplicates = [];
+
+    private $handledByPlugin = false;
+
+    /**
+     * CheckForDuplicateContactsEvent constructor.
+     */
+    public function __construct(array $fields)
+    {
+        $this->fields = $fields;
+    }
+
+    /**
+     * @return array
+     */
+    public function getFields()
```

Comment:
```suggestion
    public function getFields(): array
```
Please add param and return type for all new methods.